release-gap-package

```diff
@@ -637,12 +637,15 @@ if [ -d "$TMP_DIR/$BASENAME/htm" ] ; then
     cp -r "$TMP_DIR/$BASENAME/htm" .
 fi
 
-# adjust links to the GAP manuals
+# adjust links to the GAP manuals and to the GAPDoc manuals.
 # Note that we cannot use sed's `-i` option for in-place editing, as
 # that is a non-portable extension of POSIX, which works differently in
 # BSD and GNU make.
 for f in ./*/*.htm* ; do
-  sed 's;href="../../../doc/;href="https://www.gap-system.org/Manuals/doc/;g' "$f" > "$f.bak"
+  sed \
+    -e 's;href="../../../doc/;href="https://www.gap-system.org/Manuals/doc/;g' \
+    -e 's;href="../../../pkg/GAPDoc[^\/]*/doc/;href="http://www.math.rwth-aachen.de/~Frank.Luebeck/GAPDoc/doc/;gi' \
```

Comment:
Unfortunately this caused a regression, it breaks releases on plain macOS, where BSD sed does not support the `gi` option. Do we really need to ignore case?

Reply:
See https://github.com/gap-system/ReleaseTools/pull/87

This script badly needs a test suite :/